lib/libvcc/vcc_action.c

```diff
@@ -263,6 +263,23 @@ vcc_act_return_pass(struct vcc *tl)
 	Fb(tl, 1, ");\n");
 	tl->indent -= INDENT;
 }
+
+/*--------------------------------------------------------------------*/
+
+static void
+vcc_act_return_retry(struct vcc *tl)
+{
+	SkipToken(tl, '(');
+	if (!vcc_IdIs(tl->t, "fetch")) {
+		VSB_cat(tl->sb, "Expected: return (retry(fetch))\n");
+		vcc_ErrWhere(tl, tl->t);
+		return;
+	}
+	Fb(tl, 1, "VPI_retry_fetch(ctx);\n");
+	vcc_NextToken(tl);
+	SkipToken(tl, ')');
+}
```

Comment:
In the commit introducing `return (retry(fetch))` there should be a documentation update.